services/docker-stack-si-ps.yml

```diff
@@ -0,0 +1,2472 @@
+          # - node.hostname != IA-14RaDiag03-CO-IOCSrv
+          # - node.hostname != IA-16RaBbB-CO-IOCSrv
+          # - node.hostname != IA-18RaDiag04-CO-IOCSrv
+          # - node.hostname != IA-20RaDiag01-CO-IOCSrv-1
+          # - node.hostname != IA-20RaDiag01-CO-IOCSrv-2
+          # - node.hostname != IA-20RaDiag02-CO-IOCSrv-1
+          # - node.hostname != IA-20RaDiag02-CO-IOCSrv-2
+          - node.hostname != CA-RaCtrl-CO-Srv-1
+          - node.hostname != LA-RaCtrl-CO-Srv-1
+          - node.hostname != TA-TiRack-CO-FWSrv-1
+      replicas: 1
+      restart_policy:
+        condition: any
+    logging:
+      driver: "json-file"
+      options:
+        max-file: "10"
+        max-size: "10m"
+    networks:
+      - ioc-network
+  
+  sexts-sfa12-sda3p1-sfb0-sdb01:
+    image: dockerregistry.lnls-sirius.com.br/fac/fac-iocs:2020-03-02-test-si-corrs
+    command: bash -c 'source /etc/profile && /ioc-apps/si-ps-si-ps-sexts-sfa12-sda3p1-sfb0-sdb01.bash'
+    deploy:
+      placement:
+        constraints:
+          # - node.hostname != CA-RaTim-CO-IOCSrv
+          # - node.hostname != IA-14RaDiag03-CO-IOCSrv
+          # - node.hostname != IA-16RaBbB-CO-IOCSrv
+          # - node.hostname != IA-18RaDiag04-CO-IOCSrv
+          # - node.hostname != IA-20RaDiag01-CO-IOCSrv-1
+          # - node.hostname != IA-20RaDiag01-CO-IOCSrv-2
+          # - node.hostname != IA-20RaDiag02-CO-IOCSrv-1
+          # - node.hostname != IA-20RaDiag02-CO-IOCSrv-2
+          - node.hostname != CA-RaCtrl-CO-Srv-1
+          - node.hostname != LA-RaCtrl-CO-Srv-1
+          - node.hostname != TA-TiRack-CO-FWSrv-1
+      replicas: 1
+      restart_policy:
+        condition: any
+    logging:
+      driver: "json-file"
+      options:
+        max-file: "10"
+        max-size: "10m"
+    networks:
+      - ioc-network
+
+  sexts-sfb12-sdb3-sfp12-sdp23:
+    image: dockerregistry.lnls-sirius.com.br/fac/fac-iocs:2020-03-02-test-si-corrs
+    command: bash -c 'source /etc/profile && /ioc-apps/si-ps-si-ps-sexts-sfb12-sdb3-sfp12-sdp23.bash'
+    deploy:
+      placement:
+        constraints:
+          # - node.hostname != CA-RaTim-CO-IOCSrv
+          # - node.hostname != IA-14RaDiag03-CO-IOCSrv
+          # - node.hostname != IA-16RaBbB-CO-IOCSrv
+          # - node.hostname != IA-18RaDiag04-CO-IOCSrv
+          # - node.hostname != IA-20RaDiag01-CO-IOCSrv-1
+          # - node.hostname != IA-20RaDiag01-CO-IOCSrv-2
+          # - node.hostname != IA-20RaDiag02-CO-IOCSrv-1
+          # - node.hostname != IA-20RaDiag02-CO-IOCSrv-2
+          - node.hostname != CA-RaCtrl-CO-Srv-1
+          - node.hostname != LA-RaCtrl-CO-Srv-1
+          - node.hostname != TA-TiRack-CO-FWSrv-1
+      replicas: 1
+      restart_policy:
+        condition: any
+    logging:
+      driver: "json-file"
+      options:
+        max-file: "10"
+        max-size: "10m"
+    networks:
+      - ioc-network
+
+    sexts-sfb12-sdb3-sfp12-sdp23:
```

Comment:
is the indentation correct here?

Reply:
another duplicate here. the duplicate also has wrong indentation.